docs/sources/mobile-app/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Grafana OnCall mobile app
+aliases:
+  - /docs/oncall/latest/mobile-app/
+keywords:
+  - Mobile App
+  - oncall
+  - notification
+  - push notification
+weight: 1200
+---
+
+# Grafana OnCall Mobile App
+
+>**Note**: This application is currently in beta
+
+The Grafana OnCall mobile app allows teams to monitor and respond to critical system events from anywhere.
+On-call engineers can reimagine incident response with real-time alerts, on-call notifications, and a growing feature set.
+
+## About Grafana OnCall mobile app
+Grafana OnCall for iOS and Android allows you to:
+
+- Override Do Not Disturb on your device to ensure delivery of critical alerts
+- View and troubleshoot alert details from your mobile device
+- Receive push notifications according to your personal notification settings
+- Login with a quick and secure QR code authorization
+
+The OnCall mobile app allows users to receive push notifications as part of their notification policy.
+Push notifications are one of the many notification options in Grafana OnCall.Regardless of your notification preferences, it’s recommended that you configure a diverse set of notification methods.
+
+The current version of the mobile app includes limited functionality and serves primarily as a notification method. The mobile app is in development, and some features are not yet available.
+Stay tuned, we're working on it!
+
+## Before you begin
+
+The Grafana OnCall mobile app is intended to be used as an extension of your Grafana OnCall instance.
+Grafana OnCall is available for Grafana Cloud and Grafana open source users.
+
+- You must have a Grafana OnCall account to use this application
+- Download Grafana OnCall Mobile app
+
+Mobile app download - [Google Play](https://play.google.com/apps/testing/com.grafana.oncall.prod) and [Apple App Store](link)
+
+
+## Connect your Grafana OnCall account 
+
+The OnCall mobile app uses a QR code authentication to connect to your Grafana OnCall instance.
+You can associate one Grafana OnCall user with your OnCall mobile app.
+
+To connect your account in the mobile app:
+1. Open the Grafana OnCall mobile app
+1. Click **Sign in** and enter your Grafana Cloud credentials
```

Comment:
there are no credentials to be filled in. the QR code does all that.

Reply:
That's for clarifying, I thought from the Figma screens you also had to sign in first. Which seemed odd, so makes sense that's not the case.

Reply:
the qr code scan is effectively the sign in.  it's pretty neat (and my first time working with this process)